packages/terra-date-picker/translations/en.json

```diff
@@ -2,5 +2,6 @@
   "Terra.datePicker.dateFormat": "MM/DD/YYYY",
   "Terra.datePicker.disabled": "Disabled",
   "Terra.datePicker.openCalendar": "Open Calendar",
-  "Terra.datePicker.today": "Today"
+  "Terra.datePicker.today": "Today",
+  "Terra.datePicker.date": "Date"
```

Comment:
Was this a missing translation or do we need to add this new translation key and request translations for Data for each locale?